Plugins/Flow.Launcher.Plugin.BrowserBookmark/Main.cs

```diff
@@ -143,8 +172,8 @@ public void ReloadData()
         public static void ReloadAllBookmarks()
         {
             cachedBookmarks.Clear();
-
-            cachedBookmarks = BookmarkLoader.LoadAllBookmarks(_settings);
+            DisposeFileWatchers();
```

Comment:
is there a reason you dispose all file watchers and re-initialize them on every bookmark reload?

Reply:
if user unchecks a browser then we don't need to monitor the browser anymore.

Reply:
yeah but `ReloadAllBookmarks` will also get called when either of the monitored files changes...